standalone/src/models/custom-application/custom-application-permission/custom-application-permission-draft/presets/view-only-permissions-draft.ts

```diff
@@ -0,0 +1,33 @@
+import { entryPointUriPathToResourceAccesses } from '@commercetools-frontend/application-config/ssr';
+import camelCase from 'lodash/camelCase';
+import sampleSize from 'lodash/sampleSize';
+import upperFirst from 'lodash/upperFirst';
+import type { TBuilder } from '@/core';
+import { supportedViewOAuthScopes } from '../../../constants';
+import { CustomApplicationPermissionDraftGraphql } from '../index';
+import type { TCustomApplicationPermissionDraftGraphql } from '../types';
+
+const populatePreset = <
```

Comment:
Same as before: this seems an unnecessary abstraction since we only have one preset in this file.

Reply:
fixed in https://github.com/commercetools/test-data/pull/827/commits/27ec5be4f11caeadd0bd34690c61483654a7ed55